assignments/callbacks.js

```diff
@@ -2,33 +2,62 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
-}
+  cb(arr[0]);
+
 
+}
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
+
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[length - 1]);
+
 }
 
+
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  cb(x + y)
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  cb(x * y);
+
 }
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  if (list.includes(item) == true){
```

Comment:
`if (list.includes(item) == true) cb(true);`
`else cb(false);`

You could have done it this way, but just to be clear, some companies will require that it is written the way you did because it can prevent errors from happening if people decide to start editing the code by adding in more lines or something. So your way may be considered better.